### ISSUE TITLE
Discussion: Does DFT really have practical effects?

### ISSUE THREAD

**Issue by Kidand** [modelscope/ms-swift#5386]
I noticed that ms-swift implements DFT by adding `--enable_dft_loss true`. However, based on my practice, with all other hyperparameters remaining unchanged, merely enabling DFT results in lower loss, but the performance of the model trained with DFT is significantly inferior to that of SFT.

**Comment by MRKINKI** [comment]
I encountered the same issue. Its performance is far inferior to that of SFT , and it is even worse than the pre-trained base model.

**Comment by NicholasKX** [comment]
Stop wasting time trying; the current DFT doesn’t work for my tasks.

**Comment by vibe-viscot** [comment]
In my experiments, even when trained on mathematical datasets, DFT performs worse than SFT.

**Comment by AshOneN** [comment]
https://arxiv.org/pdf/2508.11408  This work has modified the original DFT loss.
